talisker/gunicorn.py

```diff
@@ -179,6 +194,24 @@ def init(self, parser, opts, args):
 
         return cfg
 
+    def load_config(self):
+        super(TaliskerApplication, self).load_config()
+
```

Comment:
You might be right about b). I'm just concerned that that small comment in the docs about worker_exit might be missed. What's the worst case scenario if worker_exit doesn't clean up the dir?